README.md

```diff
@@ -4,21 +4,25 @@ A UI dashboard that allows CRUD operations on Zookeeper.
 
 Requirements
 ====================
-Requires Java 7 to run.
+Requires Java 7 to run and an [installation of Maven](https://maven.apache.org/install.html) to build.
 
 Setup
 ====================
-1. mvn clean install
-2. Copy the config.cfg to the folder with the jar file. Modify it to point to the zookeeper instance. Multiple zk instances are coma separated.  eg: server1:2181,server2:2181. First server should always be the leader.
-3. Run the jar. ( nohup java -jar zkui-2.0-SNAPSHOT-jar-with-dependencies.jar & )
-4. <a href="http://localhost:9090">http://localhost:9090</a> 
+1. `git clone https://github.com/DeemOpen/zkui.git`
+2. `cd zkui`
```

Comment:
so instead of directions you can copy/paste straight into a terminal window, you'd rather have vague, imprecise directions?